Update first install command to './kowalski.py --build' (#157)
dmitryduev/kowalski: readme.md

## Spin up your own `kowalski`

> ###Cloning and Envirornment configuration

Reviewer: ```suggestion
### Cloning and Environment configuration
```

> First clone the repo and cd to the cloned directory:

Reviewer: ```suggestion
Start off by cloning the repo, then `cd` into the cloned directory:
```

```bash
git clone https://github.com/dmitryduev/kowalski.git
cd kowalski
```

> Make sure you have also a python environment which meets the requirements to run `Kowalski`:

Reviewer: ```suggestion
Make sure you have a `python` environment that meets the requirements to run `Kowalski`:
```

```bash
pip install -r requirements.txt
```

You can then use the `kowalski.py` utility to manage `Kowalski`.

> ### Setting config files

Reviewer: ```suggestion
### Setting up config files
```

You need config files in order to run `Kowalski`. You can start off by copying the default config/secrets over:

```bash
cp config.defaults.yaml config.yaml
cp docker-compose.defaults.yaml docker-compose.yaml
```

`config.yaml` contains the API and ingester configs, the `supevisord` config for the API and ingester containers,
together with all the secrets, so be careful when committing code / pushing docker images.

However, if you want to run in a production setting, be sure to modify `config.yaml` and choose strong passwords!

`docker-compose.yaml` serves as a config file for `docker-compose`, and can be used for different Kowalski deployment modes.

> Kowalski comes with several possible default `docker-compose` configs (see [below](#different-deployment-scenarios) for more info).

Reviewer: ```suggestion
Kowalski comes with several template `docker-compose` configs (see [below](#different-deployment-scenarios) for more info).
```

### Building Kowalski

> Finally, once you've set the config files, you can actually build an instance of Kowalski.

Reviewer: ```suggestion
Finally, once you've set the config files, you can build an instance of Kowalski.
```

You can do this with the following command:

```bash
./kowalski.py up --build
```

You have now successfully built a `Kowalski` instance!

> Any time you want to rebuild kowalski, you need to run the same command.

Reviewer: ```suggestion
Any time you want to rebuild `kowalski`, you need to re-run this command.
```

> Start up `Kowalski` using the default config/secrets (copying them over):

Reviewer: Add a clarification that `--build` is needed if one wants to (re)build K's containers, which includes the first invocation of the command?

Author: I tried to rewrite the readme a bit, to assume less knowledge and help explain why  the `--build` is needed sometimes. Would you be able to take a second look?
